### DISCUSSION TITLE
/usr/share/nvim/runtime/lua/vim/lsp/rpc.lua:800: Spawning language server with cmd: `{ "vscode-eslint-language-server", "--stdio" }` failed.

### DISCUSSION THREAD

**Discussion by plasmoduck** [LazyVim/LazyVim#4266] [Extra Requests]
How to fix this error I get everytime I fire up nvim?
/usr/share/nvim/runtime/lua/vim/lsp/rpc.lua:800: Spawning language server with cmd: `{ "vscode-eslint-language-server", "--stdio" }` failed. The language server is either not installed, missing from PATH, or not executable.

**Comment by leonlonsdale** [comment]
I am getting the same warning for lua-language-server

`...lar/neovim/0.10.3/share/nvim/runtime/lua/vim/lsp/rpc.lua:800: Spawning language server with cmd: `{ "lua-language-server" }` failed. The language server is either not installed, missing from PATH, or not executable.`

Works fine in my custom config, but not in LazyVim. Tried completely rm rf's of nvim, local share and local state for fresh installs - no success. Mason shows the LS is installed.

Created issue: #5281
